cylc/flow/scripts/lint.py

```diff
@@ -489,10 +491,6 @@ def main(parser: COP, options: 'Values', *targets) -> None:
                 )
                 continue
             elif not cylc8 and options.linter == 'all':
-                LOG.warn(
-                    f'{target} not a Cylc 8 workflow: '
-                    'Running style checks only...'
-                )
```

Comment:
No need for a scary warning here, the migration checks are not valid for this case, they can be silently dropped.